RecoTracker/TkTrackingRegions/interface/RectangularEtaPhiTrackingRegion.h

```diff
@@ -175,6 +175,8 @@ class RectangularEtaPhiTrackingRegion final : public TrackingRegion {
 
   TrackingRegion::Hits hits(const edm::EventSetup& es, const SeedingLayerSetsHits::SeedingLayer& layer) const override;
 
+  std::vector<bool> checkTracks(reco::TrackCollection const& InputCollection) const override;
```

Comment:
do you mind to add the comment above this function as you have in the TrackingRegion.h ?
`/// return a mask over a track collection reflecting the compatability to the region`